### ISSUE TITLE
It assumes aria-hidden=true will hide the element?

### ISSUE THREAD

**Issue by gvonkoss** [Financial-Times/o-toggle#18]

**Comment by JakeChampion** [comment]
By "hide the element" do we mean visually rather than only from the assistive technologies that may be used (Screen Readers)?

If we mean visually hidden, we should use the `hidden` attribute instead as that will visually hide the element as well as make it hidden from assistive technologies.

**Comment by gvonkoss** [comment]
My bad, I meant it doesn't do this out of the box:
``` 
[aria-hidden="true"] { 
     visibility: hidden; 
}
```
which to me implies that there is an assumption that that is implemented by the product including the component.

**Comment by JakeChampion** [comment]
I don't think we should assume that all `aria-hidden` content should be hidden from the UI. An example of a place where `aria-hidden=true` is used that we would not want to hide visually is [o-ads](https://github.com/Financial-Times/o-ads/blob/85590f849cc6dd8edcd41a0fdea73b239f428354/src/js/ad-servers/gpt.js#L249).

Perhaps we should add a BEM class that hides the element visually?

**Comment by notlee** [comment]
I've added two classes to either visually or completely hide the toggle target. This is going to be released as part of the breaking cascade. Closing to keep track of progress. See:

Related issue: https://github.com/Financial-Times/o-toggle/issues/25
Related PR: https://github.com/Financial-Times/o-toggle/pull/27
